src/renderer/components/command-palette/command-dialog.tsx

```diff
@@ -0,0 +1,88 @@
+
+import { Select } from "../select";
+import { computed, observable, toJS } from "mobx";
+import { observer } from "mobx-react";
+import React from "react";
+import { commandRegistry } from "../../../extensions/registries/command-registry";
+import { clusterStore } from "../../../common/cluster-store";
+import { workspaceStore } from "../../../common/workspace-store";
+import { CommandOverlay } from "./command-container";
+import { broadcastMessage } from "../../../common/ipc";
+import { navigate } from "../../navigation";
+import { clusterViewURL } from "../cluster-manager/cluster-view.route";
+
+@observer
+export class CommandDialog extends React.Component {
+  @observable menuIsOpen = true;
+
+  @computed get options() {
+    const context = {
+      cluster: clusterStore.active,
+      workspace: workspaceStore.currentWorkspace
+    };
+
+    return commandRegistry.getItems().filter((command) => {
+      if (command.scope === "cluster" && !clusterStore.active) {
+        return false;
+      }
+
+      if (!command.isActive) {
+        return true;
+      }
+
+      try {
+        return command.isActive(context);
+      } catch(e) {
+        console.error(e);
+
+        return false;
+      }
+    }).map((command) => {
+      return { value: command.id, label: command.title };
+    }).sort((a, b) => a.label > b.label ? 1 : -1);
```

Comment:
Would it be enough to use `sort` without comparator function? We don't using numbers here and sorting items alphabetically.

I mean replace `.sort((a, b) => a.label > b.label ? 1 : -1)` to just `.sort()`

Reply:
> Would it be enough to use `sort` without comparator function? We don't using numbers here and sorting items alphabetically.
> 
> I mean replace `.sort((a, b) => a.label > b.label ? 1 : -1)` to just `.sort()`

Plain `.sort()` does not sort by labels, or does it?

Reply:
Oh, of course, we don't sort raw strings here.